File: tic-tac-toe-core/src/main/java/by/xxx/pupil/BoardUtils.java
package by.xxx.pupil;

import by.xxx.pupil.model.Board;
import by.xxx.pupil.model.CellType;

public class BoardUtils {

    public static boolean isCellEmpty(Board board, int i, int j) {
        return CellType.EMPTY == board.getCellValue(i, j);
    }

}

File: tic-tac-toe-core/src/main/kotlin/by/xxx/pupil/BoardUtils.kt
package by.xxx.pupil

import by.xxx.pupil.model.Board
import by.xxx.pupil.model.CellType

object BoardUtils {
    @JvmStatic
    fun isCellEmpty(board: Board, i: Int, j: Int): Boolean {
        return CellType.EMPTY === board.getCellValue(i, j)
    }
}